### DISCUSSION TITLE
Algolia Multiple Indices Search Functionality

### DISCUSSION THREAD

**Discussion by Isaac-Tait** [gatsbyjs/gatsby#34689] [Help] [answered]
I have Algolia setup to search my blog posts (`content -> blog`) but I have about 30 "static" pages (`src -> pages`) that I would like to add to a separate Algolia indices. My first issue is that I have only found one article discussing this specific search scenario and the author's example was broken. 

Problem 1: All the Gatsby documentation I have found is on setting up a search of the blog only, which I already have up and running...

Problem 2: I have been playing around with GraphQL queries to populate the separate "static pages" indices much like it is done with the blog post indices. However, I cannot get a query to load the content of the "static pages" - it just returns "null". 

Has anyone implemented multiple indices Algolia search before and if so can you share how you accomplished it? Thank you in advance! Cheers 🤔

**Comment by KyleAMathews** [comment]
Perhaps ask for help here? https://github.com/algolia/gatsby-plugin-algolia

**Comment by Isaac-Tait** [comment, marked as answer]
Thank you @KyleAMathews I will give that a shot
